src/Umbraco.Cms.Api.Delivery/Querying/QueryOptionBase.cs

```diff
@@ -14,30 +14,6 @@ public abstract class QueryOptionBase
     private readonly IApiDocumentUrlService _apiDocumentUrlService;
     private readonly IVariationContextAccessor _variationContextAccessor;
 
-    [Obsolete("Please use the non-obsolete constructor. Will be removed in V17.")]
```

Comment:
Don't for got to remove unused `usings` when you've done this clean-up.  For example there will be a couple of namespaces related to the `StaticServiceProvider` we no longer need in this class, and likely others where you've done similar removal of obsolete constructors.